source/ui/CLS/CLSValueEditor.cpp

```diff
@@ -55,6 +55,11 @@ CLSValueEditor::~CLSValueEditor()
 
 }
 
+void CLSValueEditor::addLayout(QLayout *newLayout)
```

Comment:
Is having this functionality necessary? I can see it being a source of complexity/confusion down the road.